library/templates/pharmacy_list_display.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * Handles the display of weno selected pharmacies
+ *
+ * @package   OpenEMR
+ * @link      https://www.open-emr.org
+ *
+ * @author    Kofi Appiah <kkappiah@medsov.com>
+ * @copyright Copyright (c) 2023 Omega Systems Group International. <info@omegasystemsgroup.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+use OpenEMR\Common\Acl\AclMain;
+use OpenEMR\Services\PharmacyService;
```

Comment:
hi Kofi, looks like you missed a couple of these which should be
`use OpenEMR\Modules\WenoModule\Services\PharmacyService;`

nice to see you making progress!